geomstats/backend/pytorch.py

```diff
@@ -317,6 +331,7 @@ def where(*args, **kwargs):
 
 def tile(x, y):
     # TODO(johmathe): Native tile implementation
+    y = tuple([int(one_y) for one_y in y])
```

Comment:
- why need the tuple
- why need of the cast to an int?
- can't you do the cast to an int with something more backend friendly?


```
y.int() 
```

Reply:
no need of the tuple -> corrected

need to cast to an int bcs gs.maximum is sometimes used as a index for tile and outputs a torch tensor (which is what we want it to do)
cannot do .int() bcs integers are also passed as indices